Running without options causes panic

```
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]: main.parseTaskName(...) github.com/dugajean/goke/cmd/cli/main.go:32 main.main() github.com/dugajean/goke/cmd/cli/main.go:25 +0x472
```

--- a/cmd/cli/main.go
+++ b/cmd/cli/main.go
@@ -28,7 +28,7 @@ func main() {
 func parseTaskName(argIndex int) string {
 	arg := ""
 
-	if len(os.Args) > (argIndex - 1) {
+	if len(os.Args) > argIndex {
 		arg = os.Args[argIndex]
 	}
 

